src/library_wasi.js

```diff
@@ -211,6 +211,7 @@ var WasiLibrary = {
   },
   $doWritev__docs: '/** @param {number=} offset */',
   $doWritev: function(stream, iov, iovcnt, offset) {
+    {{{ from64('iov') }}}
```

Comment:
Ditto.. by the time `doWritev` is called we expect iov to already have been converted by the `__sig` of the system calls.